code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,50 @@
+/*Outer Dependencies*/
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+/*Inner Dependencies*/
+import { ReturnButton } from 'components/ReturnButton';
+
+export const MovieDetails = () => {
+  const [details, setDetails] = useState([]);
+  /*using id from the <Link> in Movie-list to make sure the fetch executes on the correct url*/
+  const { id } = useParams();
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${id}?api_key=f86cf72a5f604a49cde375a4fa2c6d61&language=en-US`
+    )
```

Comment:
Like another option - you can create a variable for the URL and then use it in fetch

Reply:
We couldn't manage to create a variable for the url needing a path. So for consistency we made them both the same. 
@Karol, what do you think we should do?